thirdparty/openldap/zimbra-openldap/rpm/SPECS/openldap.spec

```diff
@@ -1,20 +1,15 @@
 Summary:            Zimbra's openldap build
 Name:               zimbra-openldap
 Version:            VERSION
-Release:            1zimbra8.8b6ZAPPEND
+Release:            1zimbra10.0b1ZAPPEND
 License:            BSD
 Source:             %{name}-%{version}.tgz
-Patch0:             ITS5037.patch
-Patch1:             writers.patch
-Patch2:             ITS7683.patch
-Patch3:             ITS8054.patch
-Patch4:             liblmdb-soname.patch
-Patch5:             multival.patch
-Patch6:             liblmdb-keysize.patch
-Patch7:             ITS9608.patch
```

Comment:
Can you remove the dropped patches from the repo?

Reply:
Sure, will do that.